### DISCUSSION TITLE
Usage of the LEGO logo

### DISCUSSION THREAD

**Discussion by vincentbriglia** [renovatebot/renovate#13164] [Suggest an Idea]
Hi good renovate people. 

following the LEGO Group's [fair-play guidelines](https://www.lego.com/en-gb/legal/notices-and-policies/fair-play/) we unfortunately have to ask you to remove the LEGO logo from the image on the README. We sadly have to insist on this to protect our brand from being diluted, or we might lose the trademark. 

> **The LEGO Logo may not be used on an Unofficial Web Site**
The bright red LEGO logo has become one of the most recognized trademarks in the world. We have worked hard to make this logo a symbol of high quality creative products for children. The logo stands for the LEGO Group and we cannot risk allowing the distinctiveness of this symbol to be diluted. We must, therefore, insist that the LEGO logo NEVER be used on an unofficial web site.

while we love Renovate and Open Source Software in general, we hope you understand our stance. Keep up the good work!

**Comment by rarkins** [comment]
Done

**Reply by vincentbriglia** [reply]
Many thanks. 💯
